channel/src/main/java/com/synopsys/integration/alert/channel/azure/boards/distribution/AzureBoardsMessageSenderFactory.java

```diff
@@ -0,0 +1,100 @@
+import com.synopsys.integration.alert.channel.api.issue.send.IssueTrackerIssueResponseCreator;
+import com.synopsys.integration.alert.channel.api.issue.send.IssueTrackerMessageSender;
+import com.synopsys.integration.alert.channel.api.issue.send.IssueTrackerMessageSenderFactory;
+import com.synopsys.integration.alert.channel.azure.boards.AzureBoardsProperties;
+import com.synopsys.integration.alert.channel.azure.boards.AzureBoardsPropertiesFactory;
+import com.synopsys.integration.alert.channel.azure.boards.distribution.delegate.AzureBoardsIssueCommenter;
+import com.synopsys.integration.alert.channel.azure.boards.distribution.delegate.AzureBoardsIssueCreator;
+import com.synopsys.integration.alert.channel.azure.boards.distribution.delegate.AzureBoardsIssueTransitioner;
+import com.synopsys.integration.alert.channel.azure.boards.distribution.search.AzureBoardsAlertIssuePropertiesManager;
+import com.synopsys.integration.alert.common.exception.AlertException;
+import com.synopsys.integration.alert.common.persistence.model.job.details.AzureBoardsJobDetailsModel;
+import com.synopsys.integration.alert.common.rest.ProxyManager;
+import com.synopsys.integration.alert.descriptor.api.AzureBoardsChannelKey;
+import com.synopsys.integration.azure.boards.common.http.AzureApiVersionAppender;
+import com.synopsys.integration.azure.boards.common.http.AzureHttpService;
+import com.synopsys.integration.azure.boards.common.service.comment.AzureWorkItemCommentService;
+import com.synopsys.integration.azure.boards.common.service.state.AzureWorkItemTypeStateService;
+import com.synopsys.integration.azure.boards.common.service.workitem.AzureWorkItemService;
+
+@Component
+public class AzureBoardsMessageSenderFactory implements IssueTrackerMessageSenderFactory<AzureBoardsJobDetailsModel, Integer> {
+    private final Gson gson;
+    private final IssueTrackerCallbackInfoCreator callbackInfoCreator;
+    private final AzureBoardsChannelKey channelKey;
+    private final AzureBoardsPropertiesFactory azureBoardsPropertiesFactory;
+    private final ProxyManager proxyManager;
+
+    @Autowired
+    public AzureBoardsMessageSenderFactory(
+        Gson gson,
+        IssueTrackerCallbackInfoCreator callbackInfoCreator,
+        AzureBoardsChannelKey channelKey,
+        AzureBoardsPropertiesFactory azureBoardsPropertiesFactory,
+        ProxyManager proxyManager
+    ) {
+        this.gson = gson;
+        this.callbackInfoCreator = callbackInfoCreator;
+        this.channelKey = channelKey;
+        this.azureBoardsPropertiesFactory = azureBoardsPropertiesFactory;
+        this.proxyManager = proxyManager;
+    }
+
+    @Override
+    public IssueTrackerMessageSender<Integer> createMessageSender(AzureBoardsJobDetailsModel distributionDetails) throws AlertException {
+        AzureBoardsProperties azureBoardsProperties = azureBoardsPropertiesFactory.createAzureBoardsProperties();
+        String organizationName = azureBoardsProperties.getOrganizationName();
```

Comment:
Is there a reason we call this so early? Since it's a simple getter I imagine we should just call it in the return statement

Reply:
Oh yeah, this is some copy-pasta from the previous factory.